src/Components/AddMedia/AddMedia.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useRef } from "react";
+import icon from "./photo.svg";
+import styles from "./AddMedia.module.css";
+import { Media } from "..";
+
+type Props = {
+  addFile: (newMedia: Media[]) => void;
+  mediaElement: (addFiles: Media[], file: File) => void;
+};
+
+export const AddMedia: React.FC<Props> = ({ addFile, mediaElement }) => {
```

Comment:
Conversely, a component should be named for what it _is_, not what it does (see Naming things: https://docs.canva.tech/style-guides/universal/#use-a-noun-phrase-for-a-thing)
eg. `MediaUploadForm` or `MediaInput` etc.

Check your other components too